Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1375,11 +1403,32 @@ def run_polling_command(client, args: dict, cmd: str, upload_function: Callable,
             timeout_in_seconds=6000)
 
         command_result = CommandResults(scheduled_command=scheduled_command)
+    elif post_function:
```

Comment:
is the `post_function` used?

Reply:
Yes, I forgot to reference it, and instead directly referenced `validate_submit_file_polling_results`.